function-app/adb-to-purview/src/Function.Domain/Helpers/PurviewCustomType.cs

```diff
@@ -781,9 +813,12 @@ public async Task<List<Asset>> Get_Entity(string quilifiedName, string typeName)
         {
             var correlationId = Guid.NewGuid().ToString();
             var token = this.GetToken();
-            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? ""}.purview.azure.com/catalog/api";
-            var purviewSearchEndpoint = $"{baseurl}/atlas/v2/entity/bulk/uniqueAttribute/type/";
-            var deleteEndPoint = $"{baseurl}/atlas/v2/entity/guid/";
+ 
+            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? ""}.{Environment.GetEnvironmentVariable("ResourceUri") ?? "purview.azure.com"}";
```

Comment:
This breaks anyone's existing appsettings since the current default is https://purview.azure.com. Should this be a constant variable instead of the string being repeated?